average_transition_times.csv

```diff
@@ -0,0 +1,55 @@
+State_name,Species_id,Species_name,Year,Attribute,Mean_start_transition_time,Std_start_transition_time,Mean_end_transition_time,Std_end_transition_time
```

Comment:
Important - these are the column names in the mean / standard deviation transition times csv deliverable